tremor-script/src/std_lib/url.rs

```diff
@@ -0,0 +1,59 @@
+// Copyright 2018-2020, Wayfair GmbH
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use crate::registry::Registry;
+use crate::tremor_fn;
+use percent_encoding::{percent_decode_str, utf8_percent_encode, NON_ALPHANUMERIC};
+use simd_json::BorrowedValue;
+
+pub fn load(registry: &mut Registry) {
+    registry
+        .insert(tremor_fn! (url::decode(ctx, s: String) {
+            let ss = s.to_string();
```

Comment:
```percent_decode_str(&s).decode_utf8();``` should work here? `ss` assignment can be removed then.

Reply:
Well spotted. Not needed in this case